app/src/pages/authoring/[resourceType]/index.tsx

```diff
@@ -1,10 +1,35 @@
-import { Center, Text } from '@mantine/core';
+import { trpc } from '@/util/trpc';
+import { Center, Text, Divider } from '@mantine/core';
+import { useRouter } from 'next/router';
+import { ArtifactResourceType, FhirArtifact, ResourceInfo } from '@/util/types/fhir';
+import ResourceCards from '@/components/ResourceCards';
+import { extractResourceInfo } from '@/util/resourceCardUtils';
+
 export default function ResourceAuthoringPage() {
+  const router = useRouter();
+  const { resourceType } = router.query;
+
+  const resourceTypeQuery = trpc.draft.getDrafts.useQuery(resourceType as ArtifactResourceType);
+
+  const resourceCardContent: ResourceInfo[] = (resourceTypeQuery.data as FhirArtifact[])?.map(resource => {
+    return extractResourceInfo(resource);
+  });
```

Comment:
I think it makes more sense to do this:

```suggestion
  const resourceCardContent: ResourceInfo[] = (resourceTypeQuery.data ?? []).map(resource => {
    return extractResourceInfo(resource);
  });
```

And have it be empty array when data is undefined.

Also, we should probably memo-ize this, something like this:

```ts
  const {data: artifacts} = trpc.draft.getDrafts.useQuery(resourceType as ArtifactResourceType);

  const resourceCardContent = useMemo(() => {
    return (artifacts ?? []).map(a => extractResourceInfo(a));
  }, [artifacts])

```